### ISSUE TITLE
UncommentedMain does not rise violations if main method has varargs as parameters

### ISSUE THREAD

**Issue by MEZk** [checkstyle/checkstyle#2647]
**Input file: `Test.java`**
```java
public class Test {
    public static void main(String... args) {
    }
}
```

**Configuration file: `my_check.xml`**
```xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
<module name = "Checker">
    <property name="charset" value="UTF-8"/>
    <property name="severity" value="warning"/>
    <module name="TreeWalker">
       <module name="UncommentedMain"/>
    </module>
</module>
```

**Checkstyle 6.12.1 output**
```
andreiselkin@andreiselkin ~/Downloads/temp $ java -jar checkstyle-6.12.1-all.jar -c my_check.xml Test.java
Starting audit...
Audit done.
```
